larq/optimizers_v1.py

```diff
@@ -181,8 +181,8 @@ def __getattr__(self, name):
     def get_config(self):
         fp_optimizer_config = self.fp_optimizer.get_config()
         config = {
-            "threshold": self.threshold,
-            "gamma": self.gamma,
+            "threshold": tf.keras.backend.get_value(self.threshold),
+            "gamma": tf.keras.backend.get_value(self.gamma),
```

Comment:
Just to be safe, I think we should cast it to a `float` here to be inline with what [keras does](https://github.com/keras-team/keras/blob/master/keras/optimizers.py#L213):
```suggestion
            "gamma": float(tf.keras.backend.get_value(self.gamma)),
```